thinc/util.py

```diff
@@ -124,6 +137,15 @@ def is_torch_array(obj: Any) -> bool:  # pragma: no cover
         return False
 
 
+def is_torch_gpu_array(obj: Any) -> bool:  # pragma: no cover
+    if not is_torch_array(obj):
+        return False
+    elif obj.device.type == "cuda":
+        return True
+    else:
+        return False
```

Comment:
PyTorch tensors should have `is_cuda` and we can short-circuit here:

```suggestion
    return is_torch_array(obj) and obj.is_cuda
```